packages/analytics-core/src/types/page-url-enrichment.ts

```diff
@@ -0,0 +1,2 @@
+// using an empty object for now until we have actual options
+export type PageUrlEnrichmentOptions = Record<string, never>;
```

Comment:
nitpick: better to add an experimental tag since it's exported in analytics-core to avoid breaking changes although probably no one will actually use it except previous page url plugin